app/helpers/vita_min_form_builder.rb

```diff
@@ -23,9 +24,10 @@ def vita_min_field_in_label(
 
     formatted_label = label(
         method,
-        label_contents(label_text, help_text, optional: optional) + field_html,
+        label_contents(label_text, has_help_text: help_text, optional: optional) + field_html,
```

Comment:
is there a reason we don't do `!!` here like in `vita_min_state_file_select` below?